swagger.yaml

```
@@ -119,6 +118,7 @@ tags:
   - name: Cardano » Network
   - name: Cardano » Pools
   - name: Cardano » Transactions
+  - name: Cardano » Scripts
```

```suggestion
  - name: Cardano » Scripts
  - name: Cardano » Transactions
```

Keep the list alphabetically ordered .